### ISSUE TITLE
2022/09/20 議事録

### ISSUE THREAD

**Issue by agatsssssma** [ryryrymyg/kaggle_ell#104]
### 宮城
- 敵味方チームを分けずにプレイヤーを検知するyolov7を学習。['ball', 'player', 'referee', 'goalkeepers']がラベル。どうやら学習はうまくいっているっぽい。ただし、ボールが検知できない。
	- soccer netデータ落とすときの注意：touchで[train.zipとtest.zipとchallenge.zip映像を作っておく](http://train.xn--ziptest-kd4f.xn--zipchallenge-kf4k.xn--zip-j73bza2jva46as87u79cb94f/)。一緒くたにDLすると落ちる
	- soccer netは動画から画像を切り出して入っている。動画全体をpredictする実装は新たに作らないといけない
### 我妻

- yolov7にかけたビデオをbenchmark学習モデルにかけるコードを実装中。まだ未完

### 山田
- 既存code見て内容把握。
	- DFL benchmark inference + tta + fix label error（0.283）：ラベルの再付与、efficient-net-b5モデル（TIMM(pytorch)、456）
	- DFL tito + tta（0.255）：efficient-net-b5モデル（TIMM(pytorch)、456）
	- DFL Frame inference [Enlgish | 中文]（0.255）：resnetベースの回帰モデル（liner-4-output）
	- DFL benchmark inference：efficient-net-b5モデル　0.249（tensorflow、456）
- 今後の方針考えた
	- ① 強い画像分類モデル1つ作る
		- efficent-net(image-size上げる)
		- AutoKeras(kaggle環境動くのか？)
	- ② 3DCNN系のモデルを試す
		- torchのベースライン使う？
## 宿題
- チームの分類器を別に作り、それをyolov7モデルの後ろにつける：宮城
- yolov7にかけたビデオをbenchmark学習モデルにかけるコードを実装中：我妻
- 上記①②試す：山田
## 次回予定

- 9/24(土)9:00 (仮)
